config/opal_mca.m4

```diff
@@ -176,14 +179,16 @@ of type-component pairs.  For example, --enable-mca-no-build=pml-ob1])
         msg="none (dlopen disabled)"
     elif test -z "$enable_mca_dso"; then
         DSO_all=0
-        msg=default
+        DSO_accelerator=1
+        msg="accelerator framework only"
     elif test "$enable_mca_dso" = "no"; then
         DSO_all=0
         msg=none
     elif test "$enable_mca_dso" = "yes"; then
         DSO_all=1
         msg=all
     else
+        enable_mca_dso=$enable_mca_dso",accelerator"
```

Comment:
I don't think we want to always automatically add `accelerator` to whatever list the user supplies. I think that if the user supplies a list, we should do exactly that list.

Reply:
> I don't think we want to always automatically add `accelerator` to whatever list the user supplies. I think that if the user supplies a list, we should do exactly that list.

okay makes sense